armi/physics/fuelCycle/fuelHandlerInterface.py

```diff
@@ -71,31 +71,36 @@ def interactBOC(self, cycle=None):
 
         if self.enabled():
             self.manageFuel(cycle)
+        self.validateLocations()
 
     def interactEOC(self, cycle=None):
-        timeYears = self.r.p.time
+        """
+        End of cycle hook. Record cycle time and report number of assemblies in SFP.
+        """
         # keep track of the EOC time in years.
-        self.cycleTime[cycle] = timeYears
+        self.cycleTime[cycle] = self.r.p.time
         runLog.extra(
             "There are {} assemblies in the Spent Fuel Pool".format(
                 len(self.r.core.sfp)
             )
         )
 
     def interactEOL(self):
-        """Make reports at EOL"""
+        """
+        End of life hook. Generate operator life shuffle report.
+        """
         self.makeShuffleReport()
 
     def manageFuel(self, cycle):
-        """Perform the fuel management for this cycle."""
+        """
+        Perform the fuel management for a given cycle.
+        """
         fh = fuelHandlerFactory.fuelHandlerFactory(self.o)
-        fh.prepCore()
-        fh.prepShuffleMap()
-        # take note of where each assembly is located before the outage
-        # for mapping after the outage
-        self.r.core.locateAllAssemblies()
-        shuffleFactors, _ = fh.getFactorList(cycle)
-        fh.outage(shuffleFactors)  # move the assemblies around
+        fh.preoutage()
+        ## is the factor list useful at this point?
+        shuffleFactors = fh.getFactorList(cycle)
```

Comment:
The documentation on what this is seems to be missing or not clear to me at least. This appears to be used for branch searches so that a user can dynamically turn on and off the behavior of the fuel shuffling? I'd leave in for now though. @ntouran will have to help document this or clean it up.

Reply:
Will leave in for now.